dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -14,14 +14,14 @@ import 'package:intl/intl.dart' as intl;
 import 'stock_strings_en.dart';
 import 'stock_strings_es.dart';
 
-/// Callers can lookup localized strings with an instance of StockStrings returned
-/// by `StockStrings.of(context)`.
+/// Callers can lookup localized strings with an instance of StockStrings
+/// returned by `StockStrings.of(context)`.
 ///
 /// Applications need to include `StockStrings.delegate()` in their app's
-/// localizationDelegates list, and the locales they support in the app's
-/// supportedLocales list. For example:
+/// `localizationDelegates` list, and the locales they support in the app's
```

Comment:
@gmackall Does the regenerate gradle lockfiles script change dartdocs?

Reply:
Hmm, these diffs also broke analysis:

```
Analyzing flutter...                                            

   info • The expression has no effect and can be removed • dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart:69:83 • noop_primitive_operations
   info • Convert 'locale' to a super parameter • dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings_en.dart:12:3 • use_super_parameters
   info • Convert 'locale' to a super parameter • dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings_es.dart:12:3 • use_super_parameters
```

Reply:
This happened before: https://github.com/flutter/flutter/pull/137190/files#r1371888460

It's unclear to me how that script would touch dart files, though...

Reply:
Oh, we're calling `flutter pub get`, which WILL do localization code generation